screens/PaymentOrder.js

```diff
@@ -76,10 +76,13 @@ class PaymentOrder extends React.Component {
   handleSubmit = values => {
     var cashObj = {
       orderId: this.props.navigation.state.params.order.orderId,
-      paymentMethod: 'CASH',
+      paymentMethod: values.hasOwnProperty('cardNumber') ? 'CARD' : 'CASH',
```

Comment:
Should use selected payment type to determine payment method. Don't use cardNumber.